### DISCUSSION TITLE
Broken tests due to missing partial

### DISCUSSION THREAD

**Discussion by samcoppock** [wintercms/winter#1149] [General]
We have a large number of tests which worked in a previous version of winter and before that in october.

these tests test backend index/create/update/delete pages by using http calls ( $this->get() ) followed by asserts on the $response object 

$this->assetOk() and $this->assertStatus(200) is failing because its returning status 500

and the html which gets returned appears to be an error page which states things like

The partial '_field_recordfinder' is not found. (500 Internal Server Error)
The partial '_field_repeater' is not found. (500 Internal Server Error)

It should be noted that this only happens during testing the pages work fine during normal use.

is there any reason why '_field_recordfinder' and '_field_repeater' and other similar partials would now fail to be found while testing?

**Comment by LukeTowers** [comment]
@samcoppock that's very odd. Are they referencing the partials directly using the .htm extension instead of the .php extension?
